src/lua/fiber.c

```diff
@@ -563,6 +582,42 @@ lbox_fiber_status(struct lua_State *L)
 	return 1;
 }
 
+/**
+ * Get fiber info: number of context switches, backtrace, id,
+ * total memory, used memory.
+ */
+static int
+lbox_fiber_object_info(struct lua_State *L)
+{
+	struct fiber *f = get_fiber(L);
+	if (f == NULL || f->fid == 0)
```

Comment:
How can its fid be zero? Is there a test confirming this check is needed? If this is the self fiber, it is not dead by definition. If it is not self but dead, then `fiber_find()` in `get_fiber()` wouldn't find it, and `f == NULL` would be true. Do I miss something?
I see you don't do the fid check in `lbox_fiber_csw()` even though it is not any different.

Reply:
You are right. Removed extra check.